versions.json

```diff
@@ -353,6 +353,12 @@
             "npmName": "@vaadin/react-components"
         }
     },
```

Comment:
I feel this should be unified in the same block
```
  react: {
     react-components: {},
     react-components-pro: {}
  }

```

Reply:
yeah, it can be. for now, it seems flow is [using the same template](https://github.com/vaadin/flow/pull/18559/files#diff-b1223a5fd26dba93d341bdace64cadce2330f28c958ef3063d5df8a9cfd79243R87) as in this PR. 

the versions.json is only for us to record the versions and most likely to no one is going to hand update the dependency. so should not matter too much.

Reply:
now the react components split work has done. 
maybe we should ask for a new release then.